rackcity/views/report_views.py

```diff
@@ -45,6 +46,8 @@ def compute_rack_report(datacenter_id=None):
     owner_allocation = {}
     num_full_rack_slots = 0
     for asset in assets:
+        if asset.model.is_blade_asset():
+            continue
```

Comment:
I think this is unnecessary because you filtered for only assets with a rack but it's fine

Reply:
oh right hehe oh well